Bottom: c4c5f3f7819d6ac76cd6e8cedef76fc945c55712
Top:    7ea1190c757b879b46b07e30fa9e359e172aae59
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-11 13:20:34 -0700

Refresh of HandleProcessEvent

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/default-constructor/sb_debugger.py b/lldb/packages/Python/lldbsuite/test/python_api/default-constructor/sb_debugger.py
index 7dc500ceffe..e6267c8475e 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/default-constructor/sb_debugger.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/default-constructor/sb_debugger.py
@@ -21,7 +21,7 @@ def fuzz_obj(obj):
     listener = obj.GetListener()
     try:
         obj.HandleProcessEvent(lldb.SBProcess(), lldb.SBEvent(), None, None)
-    except TypeError:
+    except Exception:
         pass
     obj.CreateTargetWithFileAndTargetTriple("a.out", "A-B-C")
     obj.CreateTargetWithFileAndArch("b.out", "arm")
